pkg/handler/task.go

```diff
@@ -0,0 +1,112 @@
+package handler
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/OTumanov/go_final_project/pkg/model"
+
+	"github.com/gin-gonic/gin"
+	"github.com/sirupsen/logrus"
+)
+
+func (h *Handler) nextDate(c *gin.Context) {
+	var nd model.NextDate
+
+	if c.ShouldBindQuery(&nd) == nil {
+		logrus.Println(fmt.Sprintf(
```

Comment:
Краткость, чтобы не плодить много логов
Так же есть встроенный форматер
```suggestion
logrus.Printf("Получили объект NextDate: date: %s, now: %s, repeat: %s", nd.Date, nd.Now, nd.Repeat)
```

Reply:
Ранее с logrus не имел делал. Не увидел сразу, что он может в формат. Исправлюсь )